### DISCUSSION TITLE
Exception in callback Transaction.__retry()

### DISCUSSION THREAD

**Discussion by richlegrand** [aiortc/aiortc#1002] [Q&A]
I'm running an simple datachannel with aiortc, and I get a periodic exception: 

Exception in callback Transaction.__retry()
handle: <TimerHandle when=165025.796907407 Transaction.__retry()>
Traceback (most recent call last):
  File "/usr/lib/python3.10/asyncio/events.py", line 80, in _run
    self._context.run(self._callback, *self._args)
  File "/home/rich/.local/lib/python3.10/site-packages/aioice/stun.py", line 310, in __retry
    self.__future.set_exception(TransactionTimeout())

It's coming from the query_consent task in aioice/ice.py.  Is the fact that it's generating exceptions something to be concerned about?  It's firing off a STUN transaction every 3 seconds or so. 

thanks!

**Comment by eliabruni** [comment]
Have the same issue!

**Comment by haowendcits** [comment]
I have the same issue
